examples/moses-pln-synergy/scm/README.md

```diff
@@ -164,6 +179,10 @@ $5 = (SetLink
    )
 )
 
+### (5) Build the tautology that if X takes treatment-1, then
+    treatment-1 contains compound-A. Semi-formally
+    (\x take(x, treatment-1)) -> (\x contain(treatment-1, compound-A))
```

Comment:
I consider the meaning of a variable in Atomese/PLN to be the same as in lambda calculus. And lambda has the same meaning as well, it is an abstraction, it exposes a variable to be substituted in a body, that's all, as soon as there are no variable name clash, variables can share the same name at multiple places.